tests/container_disk_test.go

```diff
@@ -193,7 +193,7 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting from custom image location", func() {
 		Context("with disk at /custom-disk/downloaded", func() {
 			It("[test_id:1466]should boot normally", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirrosCustomLocation), "#!/bin/bash\necho 'hello'\n")
+				vmi := libvmi.NewCirros()
```

Comment:
This is using a different image: `cd.ContainerDiskCirrosCustomLocation`.
You should try `libvmi.NewCirros(WithContainerImage(cd.ContainerDiskFor(cd.ContainerDiskCirrosCustomLocation)))`.